tests/common/e2e_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 func init() {
 	testRunner = framework.E2eTestRunner
+	withAuth = WithAuth
```

Comment:
Why do we need this?

Reply:
e2e's WithAuth to override default empty one.